bve/src/parse/mesh/instructions/post_processing.rs

```diff
@@ -1,8 +1,22 @@
 use crate::parse::mesh::instructions::*;
-use crate::parse::mesh::{MeshError, MeshErrorKind, Span};
+use crate::parse::mesh::{MeshError, MeshErrorKind};
+use crate::parse::Span;
 use cgmath::{Array, Vector2, Vector3};
 use std::f32::consts::PI;
 
+/// Prepares instructions for execution.
+///
+/// Performs two postprocessing steps on the instruction list:
+///
+/// - Splits up [`Cube`] and [`Cylinder`] into their respective [`AddVertex`] and [`AddFace`] instructions. This allows
+///   the following:
+/// - Applies all [`SetTextureCoordinates`] to their vertex, moving the data into the [`AddVertex`] data structure.
+///
+/// The last bit of post processing must be done as the executor isn't actually able to edit the resulting
+/// [`Vertex`](crate::parse::mesh::Vertex) structs arbitrarily by index as [`SetTextureCoordinates`] requires.
+///
+/// Errors are taken from [`InstructionList::errors`] and any new ones encountered are appended and put in the result's
+/// [`InstructionList::errors`].
```

Comment:
I have the same questions I had for `generate_mesh` above

Reply:
I have the same answer as above.

Reply:
Resolved